gateway/src/constructors/express/routes/index.js

```diff
@@ -1,30 +1,45 @@
-const path = require('path')
-const cors = require('cors')
-const router = require('express').Router()
+const path = require("path");
+const cors = require("cors");
+const router = require("express").Router();
 
 module.exports = (context = {}) => {
-  const APIRouter = require('./api')(context)
+ const { sequelize, models } = context;
+ const APIRouter = require("./api")(context);
+ const KEYRouter = require("./key")(context);
```

Comment:
Let's have a more generic strategy for this.

- `.../api/*`
- `.../public/*`

ex: 
- `.../public/v1/apikeys/*`

Reply:
ok